agent/dockerclient/dockerapi/docker_client.go

```diff
@@ -1181,12 +1181,12 @@ func (dg *dockerGoClient) RemoveVolume(ctx context.Context, name string, timeout
 }
 
 func (dg *dockerGoClient) removeVolume(ctx context.Context, name string) error {
-	client, err := dg.dockerClient()
+	client, err := dg.sdkDockerClient()
 	if err != nil {
 		return &CannotGetDockerClientError{version: dg.version, err: err}
 	}
 
-	ok := client.RemoveVolume(name)
+	ok := client.VolumeRemove(ctx, name, false)
```

Comment:
What does the third argument indicate?

Reply:
It's means force remove: https://godoc.org/github.com/docker/docker/client#Client.VolumeRemove

The previous docker client that we use doesn't specify force remove: https://github.com/fsouza/go-dockerclient/blob/05828842f94da0aef6ad140aa3fe777d23d17139/volume.go#L123

The default is false from docker api doc:
https://docs.docker.com/engine/api/v1.36/#operation/VolumeDelete

So if the purpose is to achieve the same functionality, make it false is safe here.

Reply:
It is a placeholder for a force flag (-f) for removal of a volume. The boolean indicates whether the flag passed in or not, and with go-dockerclient, we did not pass it in.

Reply:
If removal fails for a task scoped volume, force removing it seems like the way to go. But this PR is not the right place for making this decision. We'll leave this as it is for now.

Reply:
I'll add this as a TODO to my development plan for this project.